cpp/src/arrow/pretty_print.cc

```diff
@@ -418,13 +424,16 @@ Status PrettyPrint(const ChunkedArray& chunked_arr, const PrettyPrintOptions& op
                    std::ostream* sink) {
   int num_chunks = chunked_arr.num_chunks();
   int indent = options.indent;
-  int window = options.window;
+  int window = options.container_window;
+  // Struct fields are always on new line
```

Comment:
Is there a test for that? I didn't see anything in this PR's changes, but perhaps I missed something?

Reply:
No there wasn't one. I am adding one now.